homeassistant/components/sleepiq/strings.json

```diff
@@ -23,5 +23,15 @@
         }
       }
     }
+  },
+  "selector": {
+    "foot_warmer_temp": {
+      "options": {
+        "off": "Off",
+        "low": "Low",
+        "medium": "Medium",
+        "high": "High"
+      }
+    }
```

Comment:
Did you test this? I thought it had to be entity -> select -> foot_warmer_temp instead

Reply:
That's what I originally had, but the docs say to use the selector key: https://developers.home-assistant.io/docs/internationalization/core/#selectors

Reply:
I think that's only for selectors as in selectors, not as in the select entity iirc

Reply:
Ah I see, didn't realize those were a different thing